### DISCUSSION TITLE
Per Package Usage Statistics (Installs, Updates, etc.)

### DISCUSSION THREAD

**Discussion by atteneder** [openupm/openupm#3307] [Ideas]
Hi,

First of all a massive THANK YOU for running this service. It's totally awesome ❤️

Looking at all packages on OpenUPM I see that they are sorted by popularity, which is GitHub stars count. Is there a way to figure out how often a package is/was installed (or updated)? Ideally with in-depth data like per-version numbers.

Why does that matter? Package maintainers often have to prioritize their work and having reliable usage numbers helps a lot in making fact-base decisions.

Thanks a lot!

**Comment by favoyang** [comment]
Thank you for the question. It's a frequently requested feature. Due to the limitation of verdaccio (the registry), we don't have the feature right now. I'd like to work on it when we get more resources. But no specific promise for ETA.
